### DISCUSSION TITLE
Allow regular (non-admin) user to create collections

### DISCUSSION THREAD

**Discussion by killer1loop** [directus/directus#16510] [Archive]
Is there a way to create a role that allows regular non-admin users to create and configure collections? As far as I can tell only admin users with full access to the settings can create collections, and there doesn't seem to be a way to allow access to configuring collections. I think this would be beneficial to deployments where collections are created and deleted often to avoid giving full admin permissions to users who should only be able to set up collections.

**Comment by wfriedman89** [comment]
Currently a role must have "Admin Access" enabled in order to view/edit anything inside the Settings module, which includes the Data Model 👍 Reference: https://docs.directus.io/configuration/users-roles-permissions/roles.html#configure-role-details

Additional quote from the [Settings module](https://docs.directus.io/app/settings.html) docs:

To maintain project safety and integrity, this module is only available to Administrators.

Referenced from #16727

**Comment by kasvith** [comment]
Hi

I think this need to be enabled for non admin users as well. for example in an org there are needs to give access to developers to create fields, collections without giving them permissions to add/remove users

one way to do this is adding a button here to create collections instead in settings page
<img width="288" alt="image" src="https://github.com/user-attachments/assets/5947b9ee-be44-4516-8d3b-8e7829234b48">
